tracer/test/test-applications/integrations/Samples.AWS.SQS/AsyncHelpers.cs

```diff
@@ -31,15 +31,7 @@ public static async Task SendAndReceiveMessages(AmazonSQSClient sqsClient)
                 await GetQueueUrlAsync(sqsClient);
                 await PurgeQueueAsync(sqsClient);
 
-                try
-                {
-                    await RunValidationsWithoutReceiveLoopAsync(sqsClient);
-                }
-                catch
-                {
-                    await DeleteQueueAsync(sqsClient);
```

Comment:
Why don't you want to delete the queue? Won't this mean it's not idempotent any more? 🤔

Reply:
ah yes I didn't self-comment on that one.
The initial problem that made me look at this was that exceptions were caught silently, which hinders debugging. If we want to keep that, I'd rather have it in a `finally` than a `catch`.
About losing idempotency, in a way, yes, but we start the test by purging the queue, so whether it already exists or not does not change the scenario (And creating a queue that already exists is a no-op)

Reply:
> I'd rather have it in a finally than a catch.

I agree, I actually read it as a `finally` initially 😅 

> About losing idempotency, in a way, yes, but we start the test by purging the queue, so whether it already exists or not does not change the scenario

As long as the _spans_ are the same in both cases, I'm certainly happy to leave it as-is 🙂

Reply:
yes they are the same, since the same commands are sent to AWS :)